macros/PGML.pl

```diff
@@ -274,7 +274,7 @@ sub ForceBreak {
 
 sub Par {
   my $self = shift; my $token = shift;
-  $self->End(null, shift);
+  $self->End(undef, shift);
```

Comment:
@dpvc made the clarification that he was in javascript-mode, and while passing `null` instead of `"null"` served to remedy the warning message, the appropriate choice in perl was `undef`